Mathlib/Order/Cover.lean

```diff
@@ -127,6 +127,13 @@ alias ⟨_, WCovBy.toDual⟩ := toDual_wcovBy_toDual_iff
 
 alias ⟨_, WCovBy.ofDual⟩ := ofDual_wcovBy_ofDual_iff
 
+theorem OrderIso.map_wcovBy {α β : Type*} [Preorder α] [Preorder β]
+    (f : α ≃o β) {x y : α} (h : x ⩿ y) : f x ⩿ f y := by
```

Comment:
Should be an iff. Can you make the reverse implication a lemma for order embeddings (it might already exist)?

Reply:
Done. Is the name `map_covBy` still appropriate for the iff version though?